pygeo/geo_utils/projection.py

```diff
@@ -218,12 +218,14 @@ def projectNode(pt, upVec, p0, v1, v2):
         # This just returns the two points with the minimum absolute
```

Comment:
My understanding of your changes is that `projectNode` now returns the closest point in the down direction and the closest point in the up direction instead of just the two closest points. If this is correct, could you update the code comments to clarify this?